blog/_posts/2021-03-19-tmva-pytorch.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "TMVA PyTorch Interface is Out!"
+layout: archive
+author: Anirudh Dagar
+---
+
+What if we combine [PyTorch][PyTorch] & [TMVA][TMVA]? Ever wondered how ROOT utilizes powerful external MVA libraries making them easily accessible with a direct integration into the TMVA workflow? These interfaces between TMVA & Python frameworks are powered by the PyMVA backbone. All PyMVA methods provide the same plug-and-play mechanisms as TMVA.
+
+With the release of *ROOT v6-24-00* we are excited to launch a brand new PyTorch Interface for TMVA.
+
+[PyTorch][PyTorch] is a Python-based scientific package supporting a​utomatic differentiation.​ An ​open-source machine learning​ framework that accelerates the path from research prototyping to production deployment.
+
+[PyTorch]: https://pytorch.org/
+[TMVA]: https://root.cern/manual/tmva/
+
+## Need for a PyTorch Interface?
+
+TMVA already has a PyKeras interface which we all love, especially with Keras’ simple high-level tensorflow API. If your work involves some elementary experiments, Keras maybe the goto framework due to its plug & play spirit.
+
+But things get interesting when one requires low level control and flexibility. That’s when the argument for Keras starts losing water. PyTorch on the other hand is amazing in terms of control, flexibility and raw power that it can provide to the user. It’s lower-level approach is better suited for the more mathematically-inclined users.
```

Comment:
```suggestion
But things get interesting when one requires low level control and flexibility. That’s when the argument for Keras starts losing water. PyTorch on the other hand is amazing in terms of control, flexibility and raw power that it can provide to the user. Its lower-level approach is better suited for the more mathematically-inclined users.
```